src/config/artists.rs

```diff
@@ -1,17 +1,30 @@
 use serde::{Deserialize, Serialize};
 
-#[derive(Debug, Default, Clone)]
+#[derive(Debug, Clone)]
 pub struct Artists {
     pub album_display_mode: AlbumDisplayMode,
     pub album_sort_by: AlbumSortMode,
+    pub album_date_tags: Vec<AlbumDateTag>,
 }
 
-#[derive(Debug, Default, Clone, Serialize, Deserialize, PartialEq, Eq)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub struct ArtistsFile {
     #[serde(default)]
     pub album_display_mode: AlbumDisplayMode,
     #[serde(default)]
     pub album_sort_by: AlbumSortMode,
+    #[serde(default)]
```

Comment:
this needs to keep the custom default function though unfortunatelly, because default for `Vec` is empty Vec, meaning it would change the behavior for existing configs to not use any date tag
```suggestion
    #[serde(default = "default_album_date_tags")]
```